server/cli/flags.go

```diff
@@ -105,6 +105,10 @@ var flagUsage = map[string]string{
         ranges. The scan is slowed as necessary to approximately achieve this
         duration.
 `,
+	"scan-max-idle-time": `
```

Comment:
I think it's ok for now, your quickfix is better than what we had previously, and if we remove the flag then there's still merit in it.